packages/tokamak/contracts-bedrock/deploy-config/thanos-sepolia.json

```diff
@@ -46,9 +49,20 @@
   "systemConfigStartBlock": 0,
   "requiredProtocolVersion": "0x0000000000000000000000000000000000000003000000010000000000000000",
   "recommendedProtocolVersion": "0x0000000000000000000000000000000000000003000000010000000000000000",
-  "nativeTokenName": "Tokamak Network Token",
-  "nativeTokenSymbol": "TON",
-  "nativeTokenAddress": "0xa30fe40285b8f5c0457dbc3b7c8a280373c40044",
+  "faultGameAbsolutePrestate": "0x03ab262ce124af0d5d328e09bf886a2b272fe960138115ad8b94fdc3034e3155",
+  "faultGameMaxDepth": 73,
+  "faultGameClockExtension": 10800,
+  "faultGameMaxClockDuration": 302400,
+  "faultGameGenesisBlock": ,
```

Comment:
I think we need double quota if there is no value!

Reply:
You are right! I'll update the empty values.